docs/reference/indices/delete-component-template.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[indices-delete-component-template]]
+=== Delete component template API
+++++
+<titleabbrev>Delete component template</titleabbrev>
+++++
+
+Deletes an existing component template.
+
+////
+[source,console]
+--------------------------------------------------
+PUT _component_template/template_1
+{
+  "template": {
+    "settings": {
+      "index.number_of_replicas": 0
+    }
+  }
+}
+--------------------------------------------------
+// TESTSETUP
+////
+
+[source,console]
+--------------------------------------------------
+DELETE _component_template/template_1
+--------------------------------------------------
+
+
+[[delete-component-template-api-request]]
+==== {api-request-title}
+
+`DELETE /_template/<component-template>`
+
+
+[[delete-component-template-api-desc]]
+==== {api-description-title}
+
+Use the delete component template API to delete one or more component templates
+Component templates are building blocks for constructing <<indices-templates,index templates>>  
+that specify index mappings, settings, and aliases. 
+
+[[delete-component-template-api-path-params]]
+==== {api-path-parms-title}
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index-template]
```

Comment:
This pulls in a def for `<index-template>` not `<component-template>`. I think we need a new def.

Reply:
Good catch. Will fix.